src/cargo/core/compiler/build_context/target_info.rs

```diff
@@ -74,6 +74,8 @@ pub enum FileFlavor {
     DebugInfo,
     /// SBOM (Software Bill of Materials pre-cursor) file (e.g. cargo-sbon.json).
     Sbom,
+    /// Cross-crate info JSON files generated by rustdoc.
+    DocParts,
```

Comment:
`FileFlavor::DocParts` is for filtering purpose, though I am not sure what our design principle for for adding new FileFlavor.